### PR TITLE
update to version 2.0

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Looks like there are some issues with appveyor. Could you please have a look at it.

**Comment by dougalsutherland** [comment]
@conda-forge-admin, please rerender (maybe?)

**Comment by dougalsutherland** [comment]
You know what, we can dodge this problem anyway:

@conda-forge-admin, please add noarch: python

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,33 +12,48 @@ source:
...
+    - unidecode >=0.04.14,<1.0
+    - willow >=1.1,<1.2
     - requests >=2.11.1,<3.0
    
 test:
   imports:
     - wagtail
-  commands:
-    - wagtail --help
+  requires:
```
Comment (dougalsutherland) [inline recipe/meta.yaml]: You're not actually running any tests, so you don't need to pull in all of these....

Keep the `wagtail --help` command though please.

**Comment by sannykr** [comment]
@dougalsutherland : I had removed "wagtail --help" because it didn't work. I also installed wagtail locally from pip, and tried that. It seems we can't use "wagtail --help" in wagtail2.
Should I go ahead and remove it?

**Comment by dougalsutherland** [comment]
Looks like it's `wagtail help` now; change to that. It's useful to have to make sure the CLI is installed properly.

**Pull request merged**